apps/e2e/cypress/e2e/api/artisan-commands.cy.js

```diff
@@ -0,0 +1,33 @@
+describe("Artisan command tests", () => {
+  it("Can create a new custom pool using an Artisan command", () => {
+    const uniqueTestId = (
+      Math.floor(Math.random() * Number.MAX_SAFE_INTEGER) + 1
+    ).toString();
+
+    cy.callEloquentFactory(
+      "Pool",
+      "published",
+      `{"name": {"en": "${uniqueTestId} EN", "fr": "${uniqueTestId} FR"}}`,
+    ).then((jsonObject) => {
+      // fields in JSON object from Eloquent model are snake case, not pascal case
+      cy.wrap(jsonObject.id).as("poolId");
+      cy.wrap(jsonObject.name.en).as("poolName");
+      cy.wrap(jsonObject.key_tasks.en).as("poolKeyTasks");
+    });
+
+    cy.get("@poolId").then((poolId) => {
+      cy.visit(`/en/browse/pools/${poolId}`);
+    });
```

Comment:
If this test is just to confirm that the factory created a pool correctly, it would probably be faster to confirm this with an api call, rather than a visit to a page.

Reply:
Good idea!
ref: [just use API for test](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6736/commits/63d675662c0c7f265f62fd4ea0737827eeba2b1c)